docs/instrumentations/java/java.mdx

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
When adding a new file to the docs, we also need to add it in [`mint.json`](https://github.com/odigos-io/odigos/blob/main/docs/mint.json) for it to show up in the page.

You can follow the instructions in [the README](https://github.com/odigos-io/odigos/tree/main/docs#odigos-documentation) and run mintlify tool locally to view the changes and how it will end up looking.